### PR TITLE
Automate R script 

### PR DESCRIPTION
**Pull request by glitt13**
Automate R script for calling in command line. 

## Additions

-

## Removals

-

## Changes

Can now call the following in command:
```
Rscript fs_attrs_grab.R "/path/to/attribute_config.yaml"
```

## Testing

1. No unit tests for this script because no new functions created.

## Notes

-

## Todos

-

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows project standards (link if applicable)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Target Environment support

- [ ] Windows
- [ ] Linux
- [ ] Browser

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

**Comment by glitt13** [comment]
@bolotinl , this PR assumes that the code in the fsds_fs is accepted. We will first merge any changes that happen there in case they're needed to keep this branch up-to-date.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # Regionalization and Formulation Testing and Selection (RaFTS)
 
 **Description**:  
-The formulation-selector tool, aka Regionalization and Formulation Testing & Selection (RaFTS), is under development. For more information, see the [Wiki](https://github.com/NOAA-OWP/formulation-selector/wiki). 
+The formulation-selector tool, aka Regionalation and Formulation Testing & Selection (RaFTS), is under development. For more information, see the [Wiki](https://github.com/NOAA-OWP/formulation-selector/wiki).
```
Comment (bolotinl) [inline README.md:7]: Fix spelling typo in Regionalization

File: pkg/proc.attr.hydfab/inst/extdata/xssa_attr_config_missing_vars.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ file_io: # May define {home_dir} for python's '{home_dir}/string_path'.format(ho
   - 'save_type': 'netcdf' #  Required. Use 'csv' to create a directory structure & save multiple files. May also save as hierarchical files 'netcdf' or 'zarr', or if 'csv' chosen, a directory structure is created
   - 'save_loc': 'local' #  Required.  Use 'local' for saving to a local path via dir_save. Future work will create an approach for 'aws' or other cloud saving methods
   - 'dir_base' : '{home_dir}/noaa/regionalization/data/input' # Required. The save location of standardized output
-  - 'dir_std_base' : '{dir_base}/user_data_std' # Required. The location of standardized data generated by fs_proc python package
+  - 'dir_std_base' : '{dir_base}/user_data_std' # Required. The location of standardized data generated by proc_fs python package
```
Comment (bolotinl) [inline pkg/proc.attr.hydfab/inst/extdata/xssa_attr_config_missing_vars.yaml:19]: Shouldn't this remain fs_proc?

**Review by bolotinl — DISMISSED** [review]
There are a few minor typo comments. I thought I had edited these and committed them directly myself, but now I'm not so sure those changes stuck so I added them as comments. Once these are addressed go ahead and merge.

**Comment by glitt13** [comment]
@bolotinl - resolved aforementioned mis-spellings with the latest commit, but now this needs reapproval.

**Review by bolotinl — APPROVED** [review]

**Pull request merged**